luks2.go

```diff
@@ -117,9 +117,9 @@ func (d *deviceV2) Path() string {
 func (d *deviceV2) Slots() []int {
 	var normPrio, highPrio []int
 	for i, k := range d.meta.Keyslots {
-		if k.Priority == "2" {
+		if k.Priority != nil && *k.Priority == 2 {
 			highPrio = append(highPrio, i)
-		} else {
+		} else if k.Priority == nil || *k.Priority == 1 {
```

Comment:
Could you please add a testcase for this bug to luks2_test.go? It should be a one-liner something like

```go
func TestLuks2PreferedPriority(t *testing.T) {
	runLuks2Test(t, 0, "--priority", "prefer")
}
```

Reply:
Good idea. It turns out `luksFormat` doesn't interpret the the --priority flag, but I added a test that runs an additional `config` command.